utils/build/docker/dotnet/install_ddtrace.sh

```diff
@@ -32,7 +32,7 @@ else
     tar xzf $(ls datadog-dotnet-apm*.tar.gz) -C /opt/datadog
 fi
 
-apt-get install -y binutils #we need 'strings' command to extract assembly version which is part of binutils package
+# use 'strings' command from 'binutils' package to extract assembly version
```

Comment:
[Note to reviewers] Moved `apt-get install` to Dockerfile so it can be cached in an image layer